packages/runner/package.json

```diff
@@ -13,29 +13,9 @@
     "watch": "node ../../scripts/run-webpack --watch --progress"
   },
   "devDependencies": {
-    "@cypress/react-tooltip": "0.5.3",
-    "@fontsource/mulish": "4.3.0",
-    "@fontsource/open-sans": "4.3.0",
-    "@fortawesome/fontawesome-free": "6.0.0",
-    "@packages/driver": "0.0.0-development",
-    "@packages/icons": "0.0.0-development",
-    "@packages/reporter": "0.0.0-development",
-    "@packages/web-config": "0.0.0-development",
-    "@popperjs/core": "2.9.2",
     "babel-plugin-prismjs": "1.0.2",
     "bluebird": "3.5.3",
-    "classnames": "2.3.1",
     "cross-env": "6.0.3",
-    "jquery": "3.1.1",
-    "lodash": "^4.17.21",
```

Comment:
my guess is these packages are no longer used and this is just clean up?

Reply:
Right. We migrated most of the js and ts files to other packages and they're not used any more. 

I think it will save 1 or 2 seconds when `yarn install`.